docs/source/examples/docker-containers.rst

```diff
@@ -3,16 +3,36 @@
 Using Docker Containers
 =======================
 
-SkyPilot can run a Docker container either as the runtime environment for your task, or as the task itself.
+SkyPilot can run a Docker container either as a containerized application, or the runtime environment for your task.
+
+Running Containerized Application
```

Comment:
Should we mention anything about private registries for this first case?

Reply:
for private registries with containerized apps, the user still needs to manually login first. Let me add an example 🫡